etcdctl/ctlv2/command/role_commands.go

```diff
@@ -231,6 +231,12 @@ func actionRoleGet(c *cli.Context) error {
 		os.Exit(1)
 	}
 	fmt.Printf("Role: %s\n", role.Role)
+	if "root" == role.Role {
```

Comment:
I suggest not to update v2 etcdctl anymore, it's not used actively so just keeping it would be safer.

Reply:
fine.

Reply:
Hi, The code already push, I'm not good at it, so I didn't konw the logic is right or not, need your guidance. Thanks.